birdspotter/birdspotter/settings.py

```diff
@@ -33,17 +33,27 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.getenv('DEBUG')
 CRISPY_FAIL_SILENTLY = not DEBUG
-PROD_DB = os.getenv('PROD_DB')
-PROD_FS = os.getenv('PROD_FS', 'False')
-PROD_EMAIL = os.getenv('PROD_EMAIL', 'False')
-DO_CONDA = os.getenv('DO_CONDA', 'False')
+PROD_MODE = os.getenv('PROD_MODE', 'False')
 
 TESTING = False
 TEST_RUNNER = 'birdspotter.TestRunner.TestRunner'
 
 ALLOWED_HOSTS = []
 USE_X_FORWARDED_HOST = os.getenv('USE_X_FORWARDED_HOST', 'False')
 ALLOWED_HOSTS = os.getenv('ALLOWED_HOSTS', 'localhost').split(',')
+SECURE_PROXY_SSL_HEADER = ('X-Forwarded-Proto', 'https')
+PROD_DB = os.getenv('PROD_DB')
+PROD_FS = os.getenv('PROD_FS', 'False')
+PROD_EMAIL = os.getenv('PROD_EMAIL', 'False')
+if PROD_MODE and PROD_MODE.lower() == 'true': 
```

Comment:
Shouldnt this be or?

Reply:
It's ensuring that it's not null/None before calling `.lower()` - if it's false, it'll short-circuit and skip the .lower()